packages/launchpad/cypress/e2e/scaffold-project.cy.ts

```diff
@@ -83,7 +83,11 @@ function scaffoldAndOpenCTProject (opts: {
   }
 
   cy.contains('Next Step').click()
+
+  cy.contains('Install Dev Dependencies').should('be.visible')
+  cy.contains('button', 'Waiting for you to install the dependencies...').should('be.disabled')
```

Comment:
```suggestion
  cy.contains(cy.i18n.setupWizard.installDependencies.title).should('be.visible')
  cy.contains('button', cy.i18n.setupPage.install.waitForInstall).should('be.disabled')
```

Oooh having made this suggestion, it looks like we're storing the setup install strings in two separate objects: `setupWizard` and `setupPage`. Are they both referring to the same screen? If so could be a good opportunity for a small clean up

Reply:
fyi I just made this suggestion w/o testing it, gonna pull down locally now 😅

Reply:
Alrighty confirmed it works!

Reply:
I didn't know we exposed the strings that way, interesting! I committed your suggestion and will consolidate those strings.